#! /usr/bin/env python

PACKAGE='tuw_marker_pose_estimation'
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

pose_estimation_enum = gen.enum([
    gen.const("OPENCV_ITERATIVE", int_t, 0, "OpenCV ITERATIVE"),
], "Pose estimation type")

gen.add("pose_estimation_type", int_t, 0, "Pose estimation type", 0, 0, 0, edit_method=pose_estimation_enum)
gen.add("publish_tf", bool_t, 0, "", True)
gen.add("publish_markers", bool_t, 0, "", True)

exit(gen.generate(PACKAGE, "tuw_marker_pose_estimation", "MarkerPoseEstimation"))
